### PR TITLE
Internal: Move grid container to separate context 

### PR DESCRIPTION
**Pull request by TzviRabinovitch**

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ jobs:
     needs: [build-plugin]
     strategy:
       matrix:
-        testSuite: ['elements-regression', 'default', 'nested-tabs', 'reverse-columns']
+        testSuite: ['elements-regression', 'default', 'nested-tabs', 'reverse-columns', 'containerGrid']
```
Comment (davseve) [inline .github/workflows/playwright.yml:52]: Should we maintain the convention we are currently using?
I think it should be `container-grid`

File: package.json [hunk 2]
```diff
@@ -131,4 +132,4 @@
             "react-dom": "$react-dom"
         }
     }
-}
\ No newline at end of file
+}
```
Comment (davseve) [inline package.json:135]: Please remove this empty line

File: tests/playwright/sanity/container-grid.test.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 const { test, expect } = require( '@playwright/test' );
 const WpAdminPage = require( '../pages/wp-admin-page' );
 
-test.describe( 'Container tests', () => {
+test.describe( 'Container tests @containerGrid', () => {
```
Comment (davseve) [inline tests/playwright/sanity/container-grid.test.js:4]: Same as above

File: package.json [hunk 4]
```diff
@@ -19,6 +19,7 @@
         "test:playwright:default": "npx cross-env TEST_SUITE=@default playwright test \"--config=tests/playwright/config/sanity.config.js\"",
         "test:playwright:reverse-columns": "npx cross-env TEST_SUITE=@reverse-columns playwright test \"--config=tests/playwright/config/sanity.config.js\"",
         "test:playwright:nested-tabs": "npx cross-env TEST_SUITE=@nested-tabs playwright test \"--config=tests/playwright/config/sanity.config.js\"",
+        "test:playwright:containerGrid": "npx cross-env TEST_SUITE=@ncontainerGrid playwright test \"--config=tests/playwright/config/sanity.config.js\"",
```
Comment (davseve) [inline package.json:22]: Same as above

**Review by davseve — CHANGES_REQUESTED** [review]
Playwright's container grid tests are failing.

File: package.json (hunk 2 above)
Comment (TzviRabinovitch) [reply package.json:135]: the line is added on save

**Review by davseve — APPROVED** [review]

**Pull request merged**